src/svea_sensors/launch/rtk.launch

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<!--
+    Launch RTK Localization using Ublox F9P reciever and SWEPOS Network-RTK Service
+-->
+
+<launch>
+    <!-- RTK RECEIVER ARGUMENTS -->
+    <arg name="device" default="/dev/ttyACM0"/>
+    <arg name="baud" default="250000" />
+    <arg name="gps_frame" default="gps"/>
+    <arg name="dynamic_model" default="portable"/> <!-- Allowed values are "portable", "stationary", "pedestrian","automotive","sea","airborne_1g", "airborne_2g", "airborne_4g","wrist_watch", and "bike" -->
+
+    <!-- NTRIP CLIENT ARGUMENTS -->
+    <!-- for swepos network rtk -->
+    <arg name="host" default="nrtk-swepos.lm.se"/>
+    <arg name="port" default="80"/>  <!-- PORT 8500 is also valid -->
+    <arg name="authenticate" default="true"/>
+    <arg name="mountpoint" default="MSM_GNSS"/>
+    <arg name="username" default=""/>
```

Comment:
Should username/password be optional or a requirement? It's easy to miss if they are optional... but maybe optional is better for general use ie not all NTRIP services require them?